### ISSUE TITLE
[FEATURE] Nospeedfov under viewmodel maybe

### ISSUE THREAD

**Issue by ProfKambing** [RusherDevelopment/rusherhack-issues#867]
remove the fov gained by speed

heres an example 

```


   @Shadow    public PlayerCapabilities capabilities = new PlayerCapabilities();    public float getFovModifier() {        float f = 1.0F;        if (this.capabilities.isFlying) {            f *= 1.1F;        }        IAttributeInstance iattributeinstance = this.getEntityAttribute(SharedMonsterAttributes.movementSpeed);        f = (float)((double)f * ((iattributeinstance.getAttributeValue() / (double)this.capabilities.getWalkSpeed() + 1.0D) / 2.0D));         {            f = (float)((double)f * ((iattributeinstance.getAttributeValue() / (double)this.capabilities.getWalkSpeed() + 1.0D) / 2.0D));        } else {            double d0 = 0.10000000149011612D;            if(isSprinting()) {                d0 = 0.13000000312924387D;            }            f = (float)((double)f * ((d0 / (double)this.capabilities.getWalkSpeed() + 1.0D) / 2.0D));        }        if (this.capabilities.getWalkSpeed() == 0.0F || Float.isNaN(f) || Float.isInfinite(f)) {            f = 1.0F;        }        if (this.isUsingItem() &amp;&amp; this.getItemInUse().getItem() == Items.bow) {            int i = this.getItemInUseDuration();            float f1 = (float)i / 20.0F;            if (f1 &gt; 1.0F) {                f1 = 1.0F;            } else {                f1 *= f1;            }            f *= 1.0F - f1 * 0.15F;        }        return ForgeHooksClient.getOffsetFOV((EntityPlayer) (Object) this, f);<!--EndFragment-->

```

**Comment by FluorideHistorian** [comment]
probably better for norender

**Comment by John200410** [comment]
Closed in favor of #1564
